### PR TITLE
Added Romanian Number to Words Converter.

### PR DESCRIPTION
**Pull request by cristianst85**
Added Romanian Number to Words Converter.
Fixed Romanian localization string 'roRo' -> 'roRO'.
Fixed documentation (few typos and mistakes).

Note. The Romanian converter is similar to the Italian converter, but I've made a few modifications.
Also, do I need to split this into two separated commits?

### CODE REVIEW

File: src/Humanizer/Configuration/NumberToWordsConverterRegistry.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public NumberToWordsConverterRegistry()
             Register("es", new SpanishNumberToWordsConverter());
             Register("pl", (culture) => new PolishNumberToWordsConverter(culture));
             Register("pt-BR", new BrazilianPortugueseNumberToWordsConverter());
+            Register("ro-RO", new RomanianNumberToWordsConverter());
```
Comment (hazzik) [inline src/Humanizer/Configuration/NumberToWordsConverterRegistry.cs]: Why this one is "ro-RO" and `RomanianOrdinalizer` one is just "ro"?

**Comment by cristianst85** [comment]
No reason. It can be changed to:

Register("ro", new RomanianNumberToWordsConverter());

**Comment by onovotny** [comment]
Is this PR complete and ready for review and merging?

**Comment by cristianst85** [comment]
Yes, it's complete.

**Comment by onovotny** [comment]
I've rebased this on top of the `dev` branch and merged it in, thanks for your contribution!

**Pull request closed without merging**